### PR TITLE
[#529] Fix argument names in K-Elbow docstring

### LINKED ISSUE #529: Incorrect parameter values in K-Elbow Visualizer docstring
Initially in the K-Elbow Visualizer's docstring, the possible values for the parameter `metric` are named as `distortion_score`, `silhouette_score`, and `calinski_harabaz_score`. However, using these values returns the following error:

`YellowbrickValueError: '{}' is not a defined metric use one of distortion, silhouette, or calinski_harabaz`

However, the correct names&mdash;`distortion`, `silhouette`, `calinski_harabaz`&mdash;are listed corrected further down under `Parameters`.

elbow-documentation

### PR DESCRIPTION
**Pull request by Kautumn06**
In the KElbowVisualizer's docstring, the possible values for the parameter `metric` were mistakenly named as `distortion_score`, `silhouette_score`, and `calinski_harabaz_score`. This commit updates them to the correct values: `distortion`, `silhouette`, and `calinksi_harabaz`

Fixes #529

### CODE REVIEW

**Comment by bbengfort** [comment]
@rebeccabilbro @Kautumn06 what if in the `Notes` section of the documentation we add links to the sklearn metrics? Then we can remove the `_score` suffix to avoid confusion with the parameter label, but still have the reference? 

- http://scikit-learn.org/stable/modules/generated/sklearn.metrics.calinski_harabaz_score.html 
    (https://bit.ly/2LW3Zu9) 
- http://scikit-learn.org/stable/modules/generated/sklearn.metrics.silhouette_score.html#sklearn.metrics.silhouette_score 
     (https://bit.ly/2LYWjYb)

(Probably should use the short link in the documentation)

Note that the [`distortion_score`](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/yellowbrick/cluster/elbow.py#L44) is actually implemented in Yellowbrick and should probably be added to the documentation.

**Comment by rebeccabilbro** [comment]
That sounds like a good strategy to me!

File: yellowbrick/cluster/elbow.py [hunk 1]
```diff
@@ -186,12 +186,18 @@ class KElbowVisualizer(ClusteringScoreVisualizer):
 
     If you get a visualizer that doesn't have an elbow or inflection point,
     then this method may not be working. The elbow method does not work well
-    if the data is not very clustered; in this case you might see a smooth
-    curve and the value of k is unclear. Other scoring methods such as BIC or
-    SSE also can be used to explore if clustering is a correct choice.
+    if the data is not very clustered; in this case, you might see a smooth
```
Comment (Kautumn06) [inline yellowbrick/cluster/elbow.py]: Added comma

Comment (rebeccabilbro) [reply yellowbrick/cluster/elbow.py]: Looks good!

**Comment by rebeccabilbro** [comment]
Hey there @Kautumn06 - everything looks good! I'm just going to pull your branch and build the docs locally on my machine to make sure the docstring compiles and everything, then I'll go ahead and squash/merge, ok?

**Comment by rebeccabilbro** [comment]
Okie doke @Kautumn06 - sorted! It turns out we just needed to add some indents to the seealso so that Sphinx wouldn't complain. I pushed the fix to your branch. I'll wait for the tests to pass, then merge in. Thanks for the nice contribution to this docstring to make sure users will more effectively be able to weigh their scoring options for `KElbow`!

**Pull request merged**